### DISCUSSION TITLE
Base URL / Root path for output formats other than HTML

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#2390] [Q&A]
Consider the following minimal working example:

```
---
title: "mwe"
format:
  html: default
  docx: default
---

Some text [download awesome csv](awesome.csv) more text
```

The link works for HTML but it doesn't for DOCX. Will be great to be able to define a base URL or root path to be used for relative links when converting to formats other than HTML.

Upstream issue: https://github.com/jgm/pandoc/issues/4894

**Comment by mcanouil** [comment]
Note that I cannot reproduce the issue in latest Quarto.
If the file is indeed next to the Quarto document, the link works.
